src/__tests__/index.tsx

```diff
@@ -168,9 +171,10 @@ import Demo from './demo'
     },
   }).catch(e => e)) as Error
 
-  expect(error.message).toMatchInlineSnapshot(
-    `"Could not resolve './blah-blah' from 'demo.tsx'"`,
-  )
+  expect(error.message).toMatchInlineSnapshot(`
+    "Build failed with 1 error:
+    src/index.ts:57:14: error: [inMemory] Could not resolve ./blah-blah in ./demo.tsx"
```

Comment:
Let's make sure to wrap the filepaths here in quotes if possible

Reply:
I've added quotation marks in the latest commit.